cmd/util/util.go

```diff
@@ -210,3 +215,74 @@ minikube config set WantKubectlDownloadMsg false
 			verb, fmt.Sprintf(installInstructions, constants.DefaultKubernetesVersion, goos, runtime.GOARCH))
 	}
 }
+
+type SX map[string]interface{}
+
+func StoreProcessToFile(p os.Process, filename string) error {
+	encoded := ToGOB64(map[string]interface{}{
+		filepath.Base(filename): p,
+	})
+	return ioutil.WriteFile(filename, []byte(encoded), 0644)
+}
+
+func ReadProcessFromFile(filename string) (os.Process, error) {
```

Comment:
kill has a different implementation for process for windows and unix systems:
https://golang.org/src/os/exec_windows.go
https://golang.org/src/os/exec_unix.go

when you suggest kill by pid do you have an example of how that would be done cross platform?
it seems like it would be possible with just PID but these methods do use some additional information